docs/code-style-guide.md

```diff
@@ -225,6 +225,7 @@ Some general guidelines when writing tests are:
 * in general, getter and setter methods on classes should be written like `getName()`, `setName(string name)`, with the following exceptions
   * getters with the return type `bool` may be prefixed with `is` instead of `get`, ie. `bool isActive()`
   * getters that are used _incredibly_ frequently and are _incredibly_ obvious may not require the `get` prefix. For example `Point::x()` and `Point::y()` 
+  * getters that convert units should be written as `toUnit`. For example `Angle::toDegrees()`
```

Comment:
Just double checking, is our usage of `fromType` consistent for creation as well? This should also be part of the style guide.

Reply:
Yes, but we don't have a section on constructors. Should I make one?

Reply:
I think that would be good. It seems like the ticket covers creation methods / constructors

Reply:
done